### PR TITLE
Hindi translations added

### PR DESCRIPTION
**Pull request by Anik-Bardhan**
#75 solved.
## Summary
Hindi(hi) translations added at the required places.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvements to existing code, such as refactoring or optimizations (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the [**CONTRIBUTING**](https://github.com/cicirello/.github/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/StatConfig.py [hunk 1]
```diff
@@ -241,7 +264,8 @@
     "it" : "Attività GitHub di {0}",
     "de" : "{0}s GitHub Aktivität",
     "pt" : "Atividade de {0} no GitHub",
-    "id" : "Aktivitas Github {0}"
+    "id" : "Aktivitas Github {0}",
+    "hi" : "{0} गिटहब गतिविधि",
```
Comment (JayBee007) [inline src/StatConfig.py:268]: "hi" : "{0} की गिटहब गतिविधि"

File: src/StatConfig.py [hunk 2]
```diff
@@ -232,6 +233,28 @@
...
+            "column-two" : "सभी"
+            },
+        "contributions" : {
+            "heading" : "योगदान",
+            "column-one" : "पिछला वर्ष",
+            "column-two" : "कुल"
+            },
+        "languages" : {
+            "heading" : "सार्वजनिक भंडारों में भाषा वितरण",
+            "column-one" : None,
```
Comment (JayBee007) [inline src/StatConfig.py:254]: Better to use one translation for repositories, IMHO, can just adhere to transliteration

Comment (Anik-Bardhan) [reply src/StatConfig.py:254]: changed the Hindi word for repositories in line 243. It is now same as in line 253.

File: src/StatConfig.py (hunk 1 above)
Comment (Anik-Bardhan) [reply src/StatConfig.py]: updated this on my branch.

**Review by cicirello — APPROVED** [review]

**Review by cicirello — APPROVED** [review]

**Comment by cicirello** [comment]
@Anik-Bardhan Thank you for contributing. I just merged this into main. Sometime over the next couple days I'll create a release with the new language options.

**Comment by Anik-Bardhan** [comment]
@cicirello thank you for the merge. 👍

**Comment by cicirello** [comment]
@Anik-Bardhan This just went live with release v1.9.0. :rocket:

**Pull request merged**